erin_averill/functionsBasicII.py

```diff
@@ -0,0 +1,42 @@
+# Countdown - Create a function that accepts a number as an input.  Return a new array that counts down by one, from the number 
+#(as arrays 'zero'th element) down to 0 (as the last element).  For example countDown(5) should return [5,4,3,2,1,0].
+
+def countDown(number):
+	newArr = []
+	while number >= 0:
+		newArr.append(number)
+		number = number - 1
+	return newArr
+
+countDown(5)
+
+# Print and Return - Your function will receive an array with two numbers. Print the first value, and return the second.
+
+def printReturn(arr):
+	print(arr[0])
+	return(arr[1])
+
+printReturn([1,2])
+
+# First Plus Length - Given an array, return the sum of the first value in the array, plus the array's length.
+
+def firstPlus(arr):
+	for i in arr:
```

Comment:
for i in range(0,len(arr)) //here i is the index
for i in arr //i is the values in the array